src/applications/ivc-champva/10-10D/tests/components/CustomRelationshipPattern.unit.spec.js

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai';
+import { relationshipToVeteranUI } from '../../components/CustomRelationshipPattern';
+
+describe('Custom relationship pattern', () => {
+  it('should default to veteran', async () => {
+    const relationship = relationshipToVeteranUI();
+    expect(relationship.relationshipToVeteran?.['ui:title']).to.contain(
+      'Veteran',
+    );
+  });
+});
+
+describe('Custom relationship pattern', () => {
+  it('should use functions for titles', async () => {
+    const relationship = relationshipToVeteranUI({
+      relativeTitle: () => '',
+    });
+    expect(relationship.relationshipToVeteran['ui:title']).to.exist;
+  });
+});
+
+describe('Custom relationship pattern', () => {
+  it('should allow past-tense wording', async () => {
+    const relationship = relationshipToVeteranUI({ tense: 'past' });
+    expect(relationship.relationshipToVeteran?.['ui:title']).to.contain(
+      'What’s your relationship',
+    );
+  });
+});
```

Comment:
Can this all be refactored into 1 `desrcibe` block? Or, is there a benefit I'm not seeing to  keeping these in 3 separate blocks?

Reply:
No reason for them to be separate, just didn't consider the redundancy when putting these in. [Fixed in this commit here](https://github.com/department-of-veterans-affairs/vets-website/commit/542f46c2363558913b2d39a038ef6403f745f46a)